README.md

```diff
@@ -8,6 +8,7 @@ This repository contains open source data sets.  They are intended to be used as
 
 The following folders can be found in this repository:
 
+- **[`cryptocurrency`](https://www.coingecko.com/)** - Script to pull live and historical data for specified cryptocurencies
```

Comment:
Do we want to separate samples that are just CSV/data files from samples that contain runnable code? Or is that not necessary?